plugins/modules/oneview_server_profile_template.py

```diff
@@ -212,6 +212,8 @@ def __create(self, data):
         return True, self.MSG_CREATED, resource.data
 
     def __update(self, data):
+        if "newName" in self.data:
```

Comment:
Add update example in documentation section

Reply:
done